recipes/libde265/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class Libde265Conan(ConanFile):
+    name = "libde265"
+    description = "Open h.265 video codec implementation."
+    license = "LGPL-3.0-or-later"
+    topics = ("conan", "libde265", "codec", "video", "h.265")
+    homepage = "https://github.com/strukturag/libde265"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "sse": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "sse": True
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def _patch_sources(self):
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "set(CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
```

Comment:
Is it needed?

Reply:
Yes, because:
- this recipes removes `set(CMAKE_POSITION_INDEPENDENT_CODE ON)` in upstream CMakeLists, to allow non-PIC static build.
- upstream CMakeLists files create several OBJECT targets linked into libde265 library.
- when fPIC option is removed in conanfile, conan doesn't inject any PIC logic, relying on defaut behaviour of underlying build system (for CMake: non-PIC for STATIC/OBJECT, PIC for SHARED/MODULE).

Reply:
Thanks, that's clear. Missed removal of fPIC.